opengrok-indexer/src/main/java/org/opengrok/indexer/index/Indexer.java

```diff
@@ -756,6 +756,23 @@ public static void main(String[] argv) {
                     "stylesheet. The factory-setting is: \"default\".").execute(stylePath ->
                     cfg.setWebappLAF((String) stylePath));
 
+            parser.on("--token", "=string|@file_with_string",
+                    "Authorization bearer API token to use when making API calls",
+                    "to the web application").
+                    execute(optarg -> {
+                        String value = ((String) optarg).trim();
+                        if (value.startsWith("@")) {
+                            try {
+                                String token = new String(Files.readAllBytes(Path.of(value))).trim();
```

Comment:
Is this correct handling w.r.t. the `@` sigil?

Reply:
Maybe only single line from the file should be read ?